src/quo2/components/inputs/recovery_phrase/component_spec.cljs

```diff
@@ -0,0 +1,8 @@
+(ns quo2.components.inputs.recovery-phrase.component-spec
+  (:require [quo2.components.inputs.recovery-phrase.view :as recovery-phrase]
+            [test-helpers.component :as h]))
+
+(h/describe "Recovery phrase input"
```

Comment:
Unfortunately, I couldn't manage to test the content of this input since the text inside the input are its children, It was not easy and it was taking some time, I'll be off some days, so in order to merge this I created an issue to track it: #15721